src/pybind/mgr/dashboard/frontend/src/app/core/auth/user-form/user-form.component.html

```diff
@@ -1,249 +1,190 @@
+          <span class="invalid-feedback"
+                *ngIf="userForm.showError('confirmpassword', formDir, 'required')"
+                i18n>This field is required.</span>
+        </ng-template>
+      </div>
+      <!-- Password expiration date -->
+      <div class="form-item"
+           *ngIf="!authStorageService.isSSO()">
+        <cd-date-time-picker [ngClass]="{'required': pwdExpirationSettings.pwdExpirationSpan > 0}"
+                             [control]="userForm.get('pwdExpirationDate')"
+                             name="Password expiration date"
+                             placeholder="Password expiration date"
+                             [hasTime]="false"
+                             [pwdExpirationDateValue]="pwdExpirationDateflag"
+                             i18n-name>
+        </cd-date-time-picker>
+        <span class="invalid-feedback"
+              *ngIf="userForm.showError('pwdExpirationDate', formDir, 'required')"
+              i18n>This field is required.
+        </span>
+      </div>
+      <!--Full Name-->
+      <div class="form-item">
+        <cds-text-label for="name"
+                        i18n> Full Name
+          <input cdsText
+                 type="text"
+                 placeholder="Full Name..."
+                 id="name"
+                 name="name"
+                 formControlName="name">
+        </cds-text-label>
+      </div>
+      <!-- Email -->
+      <div class="form-item">
+        <cds-text-label for="email"
+                        [invalid]="!userForm.controls.email.valid && userForm.controls.email.dirty"
+                        [invalidText]="emailError"
+                        i18n>
+          Email
+          <input cdsText
+                 type="email"
+                 placeholder="Email..."
+                 id="email"
+                 name="email"
+                 formControlName="email">
+        </cds-text-label>
+        <ng-template #emailError>
+          <span class="invalid-feedback"
+                *ngIf="userForm.showError('email', formDir, 'email')"
+                i18n>Invalid email.
+          </span>
+        </ng-template>
+      </div>
+      <!-- Roles -->
+      <div class="form-item"
+           *ngIf="allRoles">
+        <cds-combo-box label="Roles"
+                       type="multi"
+                       selectionFeedback="top-after-reopen"
+                       for="roles"
+                       name="roles"
+                       formControlName="roles"
+                       id="roles"
+                       placeholder="Select Roles..."
+                       i18n-placeholder
+                       [appendInline]="true"
+                       [items]="allRoles"
+                       itemValueKey="name"
+                       i18n>
+          <cds-dropdown-list></cds-dropdown-list>
+        </cds-combo-box>
+      </div>
+      <!-- Enabled -->
+      <div class="form-item"
```

Comment:
i'd encourage to add a new empty line before adding a new element so that the html code itself is more readable..

Reply:
changes are done